src/shared-endpoints.ts

```diff
@@ -0,0 +1,135 @@
+//type Constructor = new (...args: any[]) => {};
+import { GenericAPIResponse } from './util/requestUtils';
+import RequestWrapper from './util/requestWrapper';
+
+export default class SharedEndpoints {
+    protected requestWrapper: RequestWrapper; // XXX Is there a way to say that Base has to provide this?
+    
+    //------------Market Data Endpoints------------>
```

Comment:
It's a bit of a nitpick but can you run eslint? 
`npx eslint --fix src`

I ask as the indentation should be 2 spaces, while this and the linear client seem to have 4.

Reply:
ok

Reply:
will try later